test/Jamfile.v2

```diff
@@ -979,6 +979,7 @@ test-suite misc :
    [ run anderson_darling_test.cpp : : : [ requires cxx17_if_constexpr cxx17_std_apply ] ]
    [ run ljung_box_test.cpp : : : [ requires cxx17_if_constexpr cxx17_std_apply ] ]
    [ run test_t_test.cpp : : : [ requires cxx17_if_constexpr cxx17_std_apply ] ]
+   [ run test_z_test.cpp : : : [ requires cxx17_if_constexpr cxx17_std_apply ] ]
```

Comment:
Do you need the `requires cxx17_*` here? It looks like you've avoided type deduction using C++17 `if constexpr`.

Reply:
There is a dependency on univariate statistics which is currently C++17. Once PR #434 is merged everything I've recently edited in the stats library will drop to C++11.